src/components/TrafimageMaps/TrafimageMaps.scss

```diff
@@ -141,7 +141,7 @@
     &.header {
       .wkp-menu-wrapper {
         top: 0;
-        width: 345px;
+        width: 100vw;
```

Comment:
This one breaks the display on window's width: 960px . The menu is way too big.

Reply:
@oterral I just tested on all kinds of disply sizes and it seems to work for all cases. 100vh takes 100% of the view port width no? This was what fixed the css on the iPhone 5 display, since the menu wrapper was overflowing